### ISSUE TITLE
Window size change during record

### ISSUE THREAD

**Issue by yarona1993** [asciinema/asciinema-player#197]
When the window size change during a record, the player doesn't detect it, and the cast is not viewable
If is is possible to change the cols and rows during session? using an extra event 
like window change event

`{"version": 2, "width": 80, "height": 24, "timestamp": 1504467315, "title": "Demo", "env": {"TERM": "xterm-256color", "SHELL": "/bin/zsh"}}
[0.248848, "o", "\u001b[1;31mHello \u001b[32mWorld!\u001b[0m\n"]
[1.001376, "o", "That was ok\rThis is better."]
[1.001376, "WindowChangeEvent", [50,60]]
[2.143733, "o", " "]
[6.541828, "o", "Bye!"]
[1.001376, "WindowChangeEvent", [100,20]]
[6.541828, "i", "whoami"]
[6.541828, "o", "root"]`

**Comment by sickill** [comment]
Yeah, let's use new event type, "r" (for "resize") with "WxH" as event data:

```json
[123.45, "r", "80x24"]
```

I have an implementation of this on a local branch, will push it soon.

In order to make use of the new event we'll need to modify the player (and avt used by it).

**Comment by jonesyang12** [comment]
now(3.5.0 tag) is not supported, but playing on asciinema server can Identify resize event。When will this event be supported?

**Comment by ku1ik** [comment]
I'm going to release this today.

**Comment by ku1ik** [comment]
Just released it in [v3.6.1](https://github.com/asciinema/asciinema-player/releases/tag/v3.6.1).
